ci-operator/step-registry/baremetal/lab/upi/install/disconnected/proxy/baremetal-lab-upi-install-disconnected-proxy-workflow.yaml

```diff
@@ -4,6 +4,7 @@ workflow:
     pre:
       - chain: baremetal-lab-upi-conf-disconnected-proxy
       - chain: baremetal-lab-upi-install
+      - ref: add-redhat-registry-auth
```

Comment:
I have no much objection on this. 

The only concern is why the disconnected install run mirror images, then enable proxy together. Compared with other platform, generally if we mirrored images, then we would not enable proxy any more, also the install chain would mirror openshift imagestream in `mirror-images-tag-images`, so do not need to update cluster pull-secret. 

I would leave this to metal team to review. cc @jadhaj @mhanss @pamoedom

Reply:
Firewall is setup for disconnected environment before mirroring step.
https://github.com/openshift/release/blob/master/ci-operator/step-registry/baremetal/lab/pre/firewall/baremetal-lab-pre-firewall-commands.sh#L16

Reply:
"cluster proxy + disoconnected" v.s "mirror-images + disconnected network", I meant, if this install workflow is for "mirror-images + disconnected network", in theory, no need to enable cluster proxy. That is my concern.

Reply:
I am working on removing the cluster wide proxy in this PR #59178

Reply:
I will close this pr after https://github.com/openshift/release/pull/59178 merged

Reply:
Hi @xiuwang #59178 is merged. Can you test with the latest master?

Reply:
@sgoveas  now the cluster is a disconnected cluster, should have no such issuse.
I will get you know if I meet it again, thanks